### PR TITLE
Update new DNS

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

As per title

https://github.com/obscuronet/obscuro-internal/issues/2262

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: tools/obscuroscan/main/cli.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ type obscuroscanConfig struct {
 func defaultObscuroClientConfig() obscuroscanConfig {
 	return obscuroscanConfig{
 		nodeID:        "",
-		rpcServerAddr: "http://erpc.testnet.obscu.ro:80",
+		rpcServerAddr: "http://erpc.uat-testnet.obscu.ro:80",
```
Comment (moraygrieve) [inline tools/obscuroscan/main/cli.go:32]: not sepolia-testnet?

Comment (otherview) [reply tools/obscuroscan/main/cli.go:32]: good catch !
```suggestion
		rpcServerAddr: "http://erpc.sepolia-testnet.obscu.ro:80",
```

File: .github/workflows/manual-deploy-testnet-faucet.yml [hunk 2]
```diff
@@ -42,10 +42,10 @@ jobs:
         uses: docker/setup-buildx-action@v1
 
       - name: 'Sets env vars for testnet'
-        if: ${{ inputs.testnet_type == 'testnet' }}
+        if: ${{ inputs.testnet_type == 'uat-testnet' }}
         run: |
-          echo "FAUCET_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/faucet_testnet:latest" >> $GITHUB_ENV
-          echo "TESTNET_ADDR=erpc.testnet.obscu.ro" >> $GITHUB_ENV
+          echo "FAUCET_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/uat_faucet_testnet:latest" >> $GITHUB_ENV
```
Comment (moraygrieve) [inline .github/workflows/manual-deploy-testnet-faucet.yml:47]: not faucet_uat_testnet?

**Review by zkokelj — APPROVED** [review]
LGTM

File: .github/workflows/manual-deploy-testnet-faucet.yml (hunk 2 above)
Comment (otherview) [reply .github/workflows/manual-deploy-testnet-faucet.yml:47]: Looks like all other images are prepended at the start - should we change it ?

Comment (otherview) [reply .github/workflows/manual-deploy-testnet-faucet.yml:47]: Yeah changing it everywhere to normalize it

**Pull request merged**